coffeecard/CoffeeCard.WebApi/Services/AccountService.cs

```diff
@@ -42,6 +42,16 @@ public User GetAccountByEmail(string email)
             if (user == null) throw new ApiException("No user found with the given email", 401);
             return user;
         }
+        private User GetAccountWithTokensByEmail(string email)
+        {
+            var user = _context.Users.
+                //Include(x => x.Programme).
+                Include(x => x.Tokens).
+                FirstOrDefault(x => x.Email == email);
+
+            if (user == null) throw new ApiException("No user found with the given email", 401);
+            return user;
+        }
 
```

Comment:
Where do we use the token included in this method?

Reply:
This is the method that is used in ForgotPassword. The token is used in that one to check if there are currently any unused tokens for the user.

Reply:
I cannot see where we do any check on the tokens?

Reply:
Ahh yes my mistake. The tokens included from that user is only used for reseting them just before we save the context. This is to mark them as used in our db. The place we are checking the token itself is validateToken in tokenService

Reply:
I was just curious for if the method then is needed; can we clear the tokens without having included them in the first place? That was my thougt